### PR TITLE
Duplicate sound listing

### PR DESCRIPTION
**Pull request by DanVanAtta**
Cleanup, then remove duplicated sound list

---
This change is [Reviewable](https://reviewable.io/reviews/triplea-game/triplea/932)

### CODE REVIEW

File: src/games/strategy/sound/SoundOptions.java [hunk 1]
```diff
@@ -27,37 +27,37 @@
...
   }
 
-  public static void addToPanel(final JPanel parentPanel, final SoundType soundType) {
+  public static void addToPanel(final JPanel parentPanel) {
     final JButton soundOptions = new JButton("Sound Options...");
     soundOptions.addActionListener(new ActionListener() {
       @Override
       public void actionPerformed(final ActionEvent e) {
-        new SoundOptions(parentPanel, soundType);
+        new SoundOptions(parentPanel);
```
Comment (RoiEXLab) [inline src/games/strategy/sound/SoundOptions.java]: Here and a couple lines above lambda could easily be used...

File: src/games/strategy/sound/SoundPath.java [hunk 2]
```diff
@@ -79,63 +77,12 @@
...
-    return rVal;
+  public static Set<String> getAllSoundOptions() {
+    return getSoundOptions().stream().map(soundProp -> soundProp.getName()).collect(Collectors.toSet());
   }
 
-
-  public static ArrayList<IEditableProperty> getSoundOptions(final SoundType sounds) {
-    final ArrayList<IEditableProperty> rVal = new ArrayList<>();
+  public static List<IEditableProperty> getSoundOptions() {
+    final List<IEditableProperty> rVal = new ArrayList<>();
```
Comment (RoiEXLab) [inline src/games/strategy/sound/SoundPath.java]: rVal is probably not the best name for a variable, should be called result or something else instead...

File: src/games/strategy/sound/SoundOptions.java (hunk 1 above)
Comment (DanVanAtta) [inline src/games/strategy/sound/SoundOptions.java]: scope creep, leaving for another day.

File: src/games/strategy/sound/SoundPath.java (hunk 2 above)
Comment (DanVanAtta) [inline src/games/strategy/sound/SoundPath.java]: Took a look, the rabbit hole goes a bit deeper, would be good to fix up the SoundOptionCheckBox boiler plate code below. Changing the variable name here would blow up the diff, so I would like to keep the diff shorter and sweeter on this one.
